util/webhook/certs/README.md

```diff
@@ -0,0 +1,189 @@
+# Certificate Manager Lite
```

Comment:
Is this `Certificate Manager Lite` being used in some other components?

Reply:
This will be most likely used by addons manager as they have their own cert management solution currently. This is an attempt at unifying the code.